ntbs-service/Helpers/DisplayExtensionMethods.cs

```diff
@@ -46,7 +46,7 @@ public static string FormatYesNo(this bool? x)
         }
     }
 
-    public static class StringExtensions
+    public static partial class StringExtensions
```

Comment:
Um, I think I would just rename this class to be something else - we don't gain anything from these being partial!